lib/std/crypto/aes.zig

```diff
@@ -138,9 +146,48 @@ fn AES(comptime keysize: usize) type {
         pub fn decrypt(ctx: Self, dst: []u8, src: []const u8) void {
             decryptBlock(ctx.dec[0..], dst, src);
         }
+        pub fn ctr(ctx: Self, dst: []u8, src: []const u8, iv: [16]u8) void {
```

Comment:
might be more convenient for this function to return the new counter value; otherwise the caller has to increment it when making successive calls.